src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBreakpointEvidenceSpark.java

```diff
@@ -695,7 +724,56 @@ public AlignedAssemblyOrExcuse apply( final Tuple2<Integer, List<SVFastqUtils.Fa
             intervals.add(evidenceIterator2.next().getLocation());
         }
 
-        return intervals;
+        return new Tuple2<>(intervals, evidenceTargetLinks);
+    }
+
+    private static void writeTargetLinks(final FindBreakpointEvidenceSparkArgumentCollection params, final Broadcast<ReadMetadata> broadcastMetadata, final List<EvidenceTargetLink> targetLinks) {
+        if ( params.targetLinkFile != null ) {
+            try (final OutputStreamWriter writer =
+                         new OutputStreamWriter(new BufferedOutputStream(BucketUtils.createFile(params.targetLinkFile)))) {
+                targetLinks.iterator().forEachRemaining(entry -> {
+                    final String bedpeRecord = entry.toBedpeString(broadcastMetadata.getValue());
+                    try {
+                        writer.write(bedpeRecord + "\n");
+                    } catch (final IOException ioe) {
+                        throw new GATKException("Can't write target links to "+params.targetLinkFile, ioe);
+                    }
+                });
+            } catch ( final IOException ioe ) {
+                throw new GATKException("Can't write target links to "+params.targetLinkFile, ioe);
+            }
+        }
+    }
+
+
+    private static List<BreakpointEvidence> collectAllEvidence( final JavaRDD<BreakpointEvidence> evidenceRDD,
```

Comment:
It appears that this is not called.  Should it have been?

Reply:
No, sorry. Dev code that I mistakenly left in. Removed.